src/Microsoft.Azure.WebJobs.Host/TypeUtility.cs

```diff
@@ -116,5 +118,20 @@ public static bool IsAsyncVoid(MethodInfo methodInfo)
         {
             return IsAsync(methodInfo) && (methodInfo.ReturnType == typeof(void));
         }
+                
+        private static bool HasConverterWorker<TAttribute, TSrc, TDest>(IConverterManager converterManager)
```

Comment:
I agree the naming is a little strange. These methods don't seem like they belong on TypeUtility at all. I think they should be non-generic extension methods to IConverterManager.